docs/merge-sections.html

```diff
@@ -3,5 +3,39 @@
 title: Merge Sections
 ---
 
-<h1 class="mt-5">Merge Sections Documentation</h1>
-<p>Follow documentation for this Feature</p>
\ No newline at end of file
+<h1 class="mt-5">Merge Sections</h1>
+<p>
+  Merge Sections makes it quick and easy to combine sections from multiple Canvas courses in the same term into a single course. 
+  For example, an instructor may want to combine lab/discussion sections with the lecture section(s) into a single course for a specific term.
+</p>
+<h2 class="h5">Who Can Use Merge Sections</h2>
+<ul>
+  <li>Root Account Administrator</li>
+  <li>Sub-Account Administrator</li>
+  <li>Support Consultant</li>
+  <li>Teachers</li>
+</ul>
+<h2 class="h5">User Role limitations</h2>
+<ul>
+  <li>Sub-account administrators can merge any sections in their assigned subaccount(s) only.</li>
+  <li>Teachers can merge sections only where they have the Teacher role. </li>
+</ul>
+<h2 class="h5">How to Merge Sections</h2>
+<ol>
+  <li>Be sure you are in the Canvas course that you want to merge sections into.</li>
+  <li>Click the Merge Sections button on the Canvas Course Manager landing page.</li>
+  <li>There are two panes in the Merge Sections tool. The right side pane shows sections already in the current course and will be updated to reflect new sections as you add them.</li>
+  <li>At the top of the left pane, enter the course name to merge in the search box.
+   <br> <b>Tip:</b> When working with large numbers of lab/discussion sections, search results are most efficient when you include the course number (e.g. STATS 250, not just STATS). </li>
+  <li>Select the sections to merge by clicking on individual tiles or using the Select All box. 
+    <br> <b>Tip:</b> Use the Sort button to re-order the sections to merge by enrollments or alphabetical order.</li>
+  <li>To remove sections from the right side pane, click the desired section tile, then click the undo button.</li>
```

Comment:
I think there might be a missing step here; doesn't the user need to click "Add"?